src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,28 @@
+.catalog {
+  display: grid;
+  gap: $gap;
+  grid-template-columns: $cardWidth;
+  justify-content: center;
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, $cardWidth);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, $cardWidth);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, $cardWidth);
+  }
+
+  &__card {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: $cardWidth;
```

Comment:
You don't need to set the width for the card cause it is already set for the column.